### ISSUE TITLE
wget cmd silently finishes without any files downloaded

### ISSUE THREAD

**Issue by hernanmd** [pharo-project/pharo-zeroconf#13]
Using MSYS in Windows 10 the command

`wget -O- https://get.pharo.org | bash`

or

`wget -O- --no-check-certificate https://get.pharo.org | bash`

Does not download any file. The output is:

```
$ wget -O- --no-check-certificate https://get.pharo.org | bash
--2018-02-28 21:28:14--  https://get.pharo.org/
Resolving get.pharo.org... 164.132.235.17
Connecting to get.pharo.org|164.132.235.17|:443... connected.
WARNING: cannot verify get.pharo.org's certificate, issued by `/C=GB/ST=Greater Manchester/L=Salford/O=COMODO CA Limited/CN=COMODO RSA Domain Validation Secure Server CA':
  Self-signed certificate encountered.
WARNING: certificate common name `cluster023.hosting.ovh.net' doesn't match requested host name `get.pharo.org'.
HTTP request sent, awaiting response... 200 OK
Length: 15652 (15K) [text/html]
Saving to: `STDOUT'

100%[===========================================================================================================>] 15,652      59.2K/s   in 0.3s

2018-02-28 21:28:16 (59.2 KB/s) - written to stdout [15652/15652]


```

Command was tested with no antivirus installed and firewall disabled.

$ wget --version
GNU Wget 1.12 built on msys.

$ echo $0
bash

**Comment by MarcusDenker** [comment]
yes, this is a problem... but --no-check-certificate  kind of defeats the whole reason of ssl.

I will check if we can change the setup at the provider.

**Comment by demarey** [comment]
On OS X, it works as expected:
```bash
wget -O- --no-check-certificate https://get.pharo.org | bash
--2018-03-05 09:43:49--  https://get.pharo.org/
Resolving get.pharo.org... 164.132.235.17, 2001:41d0:301::23
Connecting to get.pharo.org|164.132.235.17|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 15652 (15K) [text/html]
Saving to: 'STDOUT'

-                                       100%[===============================================================================>]  15.29K  --.-KB/s    in 0.009s  

2018-03-05 09:43:49 (1.61 MB/s) - written to stdout [15652/15652]

Downloading the latest 61 Image:
    http://files.pharo.org/get-files/61/pharo.zip
Pharo.image
Downloading the latest pharoVM:
	http://files.pharo.org/get-files/61/pharo-mac-stable.zip
pharo-vm/Pharo.app/Contents/MacOS/Pharo
Downloading PharoV61.sources:
	http://files.pharo.org/get-files/61/sources.zip
Creating starter scripts pharo and pharo-ui
```
And with certificate check:
```bash
wget -O- https://get.pharo.org | bash
--2018-03-05 09:43:23--  https://get.pharo.org/
Resolving get.pharo.org... 164.132.235.17, 2001:41d0:301::23
Connecting to get.pharo.org|164.132.235.17|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 15652 (15K) [text/html]
Saving to: 'STDOUT'

-                                       100%[===============================================================================>]  15.29K  --.-KB/s    in 0.009s  

2018-03-05 09:43:23 (1.60 MB/s) - written to stdout [15652/15652]

Downloading the latest 61 Image:
    http://files.pharo.org/get-files/61/pharo.zip
Pharo.image
Downloading the latest pharoVM:
	http://files.pharo.org/get-files/61/pharo-mac-stable.zip
pharo-vm/Pharo.app/Contents/MacOS/Pharo
Downloading PharoV61.sources:
	http://files.pharo.org/get-files/61/sources.zip
Creating starter scripts pharo and pharo-ui
```

**Comment by MarcusDenker** [comment]
For me not:

MACBOOKAIR-9AC0:Desktop denker$ wget -O- https://get.pharo.org | bash
--2018-03-05 10:01:14--  https://get.pharo.org/
Resolving get.pharo.org... 164.132.235.17
Connecting to get.pharo.org|164.132.235.17|:443... connected.
ERROR: no certificate subject alternative name matches
	requested host name `get.pharo.org'.
To connect to get.pharo.org insecurely, use `--no-check-certificate'.

**Comment by PharoProject** [comment]
For me curl feels better than wget (which I had to install with brew), but they both work:

$ curl https://get.pharo.org | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 15652  100 15652    0     0  78808      0 --:--:-- --:--:-- --:--:-- 79050
Downloading the latest 61 Image:
    http://files.pharo.org/get-files/61/pharo.zip
Pharo.image
Downloading the latest pharoVM:
	http://files.pharo.org/get-files/61/pharo-mac-stable.zip
pharo-vm/Pharo.app/Contents/MacOS/Pharo
Downloading PharoV61.sources:
	http://files.pharo.org/get-files/61/sources.zip
Creating starter scripts pharo and pharo-ui

$ wget -O- https://get.pharo.org | bash
--2018-03-05 10:06:58--  https://get.pharo.org/
Resolving get.pharo.org (get.pharo.org)... 164.132.235.17
Connecting to get.pharo.org (get.pharo.org)|164.132.235.17|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 15652 (15K) [text/html]
Saving to: 'STDOUT'

-                       100%[===============================>]  15.29K  --.-KB/s    in 0.02s   

2018-03-05 10:06:59 (984 KB/s) - written to stdout [15652/15652]

Downloading the latest 61 Image:
    http://files.pharo.org/get-files/61/pharo.zip
Pharo.image
Downloading the latest pharoVM:
	http://files.pharo.org/get-files/61/pharo-mac-stable.zip
pharo-vm/Pharo.app/Contents/MacOS/Pharo
Downloading PharoV61.sources:
	http://files.pharo.org/get-files/61/sources.zip
Creating starter scripts pharo and pharo-ui

$ sw_vers 
ProductName:	Mac OS X
ProductVersion:	10.13.3
BuildVersion:	17D102

$ curl --version
curl 7.54.0 (x86_64-apple-darwin17.0) libcurl/7.54.0 LibreSSL/2.0.20 zlib/1.2.11 nghttp2/1.24.0
Protocols: dict file ftp ftps gopher http https imap imaps ldap ldaps pop3 pop3s rtsp smb smbs smtp smtps telnet tftp 
Features: AsynchDNS IPv6 Largefile GSS-API Kerberos SPNEGO NTLM NTLM_WB SSL libz HTTP2 UnixSockets HTTPS-proxy 

$ wget --version
GNU Wget 1.19.4 built on darwin17.3.0.

-cares +digest -gpgme +https +ipv6 +iri +large-file -metalink +nls 
+ntlm +opie -psl +ssl/openssl 

Wgetrc: 
    /usr/local/etc/wgetrc (system)
Locale: 
    /usr/local/Cellar/wget/1.19.4_1/share/locale 
Compile: 
    clang -DHAVE_CONFIG_H -DSYSTEM_WGETRC="/usr/local/etc/wgetrc" 
    -DLOCALEDIR="/usr/local/Cellar/wget/1.19.4_1/share/locale" -I. 
    -I../lib -I../lib -I/usr/local/opt/openssl/include -DNDEBUG 
Link: 
    clang -DNDEBUG -lidn2 -L/usr/local/opt/openssl/lib -lssl -lcrypto 
    -ldl -lz ftp-opie.o openssl.o http-ntlm.o ../lib/libgnu.a -liconv 
    -lintl -Wl,-framework -Wl,CoreFoundation -lunistring 

Copyright (C) 2015 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later
<http://www.gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Originally written by Hrvoje Niksic <hniksic@xemacs.org>.
Please send bug reports and questions to <bug-wget@gnu.org>.

**Comment by hernanmd** [comment]
It would help if there is a way to disable the --quiet parameter in line 18 of view-source:http://get.pharo.org/:

`DOWNLOAD_TO="wget --quiet --output-document=";`

**Comment by hernanmd** [comment]
I can also reproduce it in a Windows 8.1

```bash
$ wget -O- https://get.pharo.org/ | bash
--2018-07-19 20:58:27--  https://get.pharo.org/
Resolving get.pharo.org... 164.132.235.17
Connecting to get.pharo.org|164.132.235.17|:443... connected.
ERROR: cannot verify get.pharo.orgâ?Ts certificate, issued by â?o/C=GB/ST=Greate
r Manchester/L=Salford/O=COMODO CA Limited/CN=COMODO RSA Domain Validation Secur
e Server CAâ??:
  Self-signed certificate encountered.
ERROR: certificate common name â?ocluster023.hosting.ovh.netâ?? doesnâ?Tt match
requested host name â?oget.pharo.orgâ??.
To connect to get.pharo.org insecurely, use â?~--no-check-certificateâ?T.
```
